utilities/transactions/optimistic_transaction_test.cc

```diff
@@ -276,52 +264,48 @@ TEST_P(OptimisticTransactionTest, FlushTest2) {
   WriteOptions write_options;
   ReadOptions read_options, snapshot_read_options;
   string value;
-  Status s;
 
-  txn_db->Put(write_options, Slice("foo"), Slice("bar"));
-  txn_db->Put(write_options, Slice("foo2"), Slice("bar"));
+  ASSERT_OK(txn_db->Put(write_options, Slice("foo"), Slice("bar")));
+  ASSERT_OK(txn_db->Put(write_options, Slice("foo2"), Slice("bar")));
 
   Transaction* txn = txn_db->BeginTransaction(write_options);
   ASSERT_TRUE(txn);
```

Comment:
Shouldn't this ASSERT_TRUE be ASSERT_NE(nullptr) ?

Reply:
Ideally yes. I have fixed that now.